docs/howto/upgrade-cluster/k8s-version-skew.md

```diff
@@ -0,0 +1,21 @@
+(upgrade-cluster:k8s-version-skew)=
+# About Kubernetes' version skew policy
+
+When we upgrade our Kubernetes clusters, we upgrade the k8s control plane (k8s
+api-server etc.), the cloud providers managed workloads (`calico-node` etc.),
+and the k8s node's software (`kubelet` etc.). Since these upgrades aren't done
+at the exact same time, there are known constraints on how the various versions
+can _skew_ in relation to each other.
```

Comment:
Is there a link to the definition of "skew" in this context we can provide? I'm not fully aware of the concept.

Reply:
I didn't find a clear definition for exaple on wikipedia, I linked out to [a blog post](https://www.industrialempathy.com/posts/version-skew/) describing it though - I haven't seen it outside kubernetes context.

Reply:
I'll come back to rephase this to not need a link to a blog post - i have to take care of a bay now though